doc-content/enterprise-only/getting-started/test-speed-scenarios-con.adoc

```diff
@@ -0,0 +1,8 @@
+[id='test-scenarios-con']
+= Test Scenarios
+
+Test Scenarios in {PRODUCT} enable you to validate the functionality of rules, models, and events before deploying them into production. A test scenario uses data for conditions that resemble an instance of your fact or project model. This data is matched against a given set of rules and if the expected results match the actual results, the test is successful. If the expected results do not match the actual results, then the test fails.
```

Comment:
replace given with specific or delete. IBM SG: given adj
Do not use to mean “particular” or “specific.” Consider whether any adjective is necessary.

Reply:
changed to "specific"